src/components/application_manager/test/hmi_capabilities.json

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
Why was this file added and why wasn't it needed before?

Reply:
This file was added by mistake during resolving of merge conflicts. Now this file is absent on develop branch, so we don't need it. Deleted in d663395